final_prep/mod_zero_hero.rb

```diff
@@ -56,16 +96,85 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = { "name" => "Chibi the Crusher",
+                  "smell" => "wet dog",
+                  "weight" => 1392,
+                  "citiesDestroyed" => ["Honolulu", "Miami", "Cincinatti"],
+                  "luckyNumbers" => [5, 22, 36],
+                  "address" => { "number" => 1108,
+                                 "street" => "Ichabod",
+                                 "state" => "Alaska",
+                                 "zip" => 99705
+                  }
+}
+ puts scary_monster
+
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+# Object and Class variables (dynamic and static)
+
+class SuperHero
+  attr_accessor :name, :super_power, :age
+
+  def initialize(name, super_power, age)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @power_level = 100
+    @energy_level = 50
+    @arch_nemesis = "The Syntax Error"
+  end
+
+  def arch_nemesis
+    puts @arch_nemesis
+  end
+
+  def power_level
+    puts @power_level
+  end
+
+  def energy_level
+    puts @energy_level
+  end
+
+  def say_name
+    puts name
+  end
+
+  def maximize_energy
+    @energy_level = 1000
+    puts @energy_level
+  end
+
+  def gain_power(boost_power)
+    @power_level += boost_power
```

Comment:
Nice dynamic methods! You don't need a puts statement at the end of all of these... it is nice to see the result of what has happened, but sometimes, you will just care that the update happens, and not need any report back to the console.